### ISSUE TITLE
What I want to do is to replicate this functionality : export to excel. But So far I haven't been able to achieve the same for pdf export in kendo angular grid . There are only two options [allPages], but I need the actual (pdf export ) event or some function in which I can provide some the state (criteria to my API) for data to be exported.

### ISSUE THREAD

**Issue by AK1510** [telerik/kendo-angular#1089]
## I'm submitting a...

* Documentation issue or request

## Current behavior

By default, the Grid exports only the current page of data. To export all pages, set the allPages option to true

## Expected behavior

What I want to do is to replicate this functionality : export to excel. But So far I haven't been able to achieve the same for pdf export in kendo angular grid .
There are only two options [allPages], but I need the actual (pdf export ) event or some function in which I can provide some the state (criteria to my API) for data to be exported.

In short this is what Iwant  : To export data that is different from the current Grid data, by specifying  a custom fetchData function.

## Environment

Package versions:

```

```

Browser:

- Chrome (desktop) version XX
- Chrome (Android) version XX
- Chrome (iOS) version XX
- Firefox version XX
- Safari (desktop) version XX
- Safari (iOS) version XX
- IE version XX
- Edge version XX

System:
- TypeScript version: XX 
- Node version: XX  
- Platform:

**Comment by AK1510** [comment]
I have seen the export method in the documentation [here ](https://www.telerik.com/kendo-angular-ui/components/grid/api/PDFComponent/) but the explination isn't that clear : 
export
Exports the content as a Group for further processing

**Comment by danielkaradachki** [comment]
This is not supported by the pdf export. You may consider opening a feature request in our [user voice forum](http://kendoui-feedback.telerik.com/forums/555517-kendo-ui-for-angular-feedback?category_id=181393).

Also, please note that this issue tracker is for bug reports only. Assistance with specific scenarios can be obtained through the support system or on community sites like StackOverflow.
